packages/terra-folder-tree/src/subcomponents/FolderTreeItem.jsx

```diff
@@ -133,16 +133,23 @@ const FolderTreeItem = ({
     ? <IconCaretDown height="8px" width="8px" style={{ verticalAlign: 'baseline' }} /> // eslint-disable-line react/forbid-component-props
     : <IconCaretRight height="8px" width="8px" style={{ verticalAlign: 'baseline' }} />; // eslint-disable-line react/forbid-component-props
 
+  /* eslint-disable jsx-a11y/click-events-have-key-events, jsx-a11y/no-noninteractive-element-interactions, jsx-a11y/label-has-associated-control */
```

Comment:
Trying to see if there is any change to avoid disabling these. For now these rules are all being disabled because:
1. jsx-a11y/click-events-have-key-events - the onClick is only used to stop propagation and prevent expand/collapse, which is not an issue for the keyboard interactions
2. jsx-a11y/no-noninteractive-element-interactions - this is just unavoidable as long as there is an onClick on a label.
3. jsx-a11y/label-has-associated-control - not sure if a bug but in theory nesting label -> input the way is being done here should satisfy this rule.